hosting/templates/hosting/order_confirm.html

```diff
@@ -0,0 +1,144 @@
+                                <span class="pull-right">{{vm.cores|floatformat}}</span>
+                            {% else %}
+                                <span class="pull-right">{{vm.cpu|floatformat}}</span>
+                            {% endif %}
+                        </p>
+                        <p>
+                            <span>{% trans "Memory" %}: </span>
+                            <span class="pull-right">{{vm.memory}} GB</span>
+                        </p>
+                        <p>
+                            <span>{% trans "Disk space" %}: </span>
+                            <span class="pull-right">{{vm.disk_size}} GB</span>
+                        </p>
+                        <p>
+                            <span>{% trans "Total" %}</span>
+                            <span class="pull-right">{{vm.price}} CHF</span>
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <hr>
+        </div>
+
+        {% block submit_btn %}
+            <form method="post" id="virtual_machine_create_form">
+                {% csrf_token %}
+                <div class="row">
+                    <div class="col-sm-8">
+                        <div class="dcl-place-order-text">{% blocktrans with vm_price=request.session.specs.price %}By clicking "Place order" this plan will charge your credit card account with the fee of {{ vm_price }}CHF/month{% endblocktrans %}.</div>
+                    </div>
+                    <div class="col-sm-4 order-confirm-btn text-right">
+                        <button class="btn choice-btn" id="btn-create-vm" data-href="{% url 'hosting:order-confirmation' %}" data-toggle="modal" data-target="#createvm-modal">
+                            {% trans "Place order" %}
+                        </button>
+                    </div>
+                </div>
+            </form>
+        {% endblock submit_btn %}
+    {% endif %}
+</div>
+
+
+<!-- Create VM Modal -->
+<div class="modal fade" id="createvm-modal" tabindex="-1" role="dialog"
+     aria-hidden="true" data-backdrop="static" data-keyboard="false">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+            </div>
+            <div class="modal-body">
+                <div class="modal-icon">
+                    <i class="fa fa-cog fa-spin fa-3x fa-fw"></i>
+                    <span class="sr-only">{% trans "Processing..." %}</span>
+                </div>
+                <h4 class="modal-title" id="createvm-modal-title"></h4>
+                <div class="modal-text" id="createvm-modal-body">
+                    {% trans "Hold tight, we are processing your request" %}
+                </div>
+                <div class="modal-footer">
+                    <a id="createvm-modal-done-btn" class="btn btn-success btn-ok btn-wide hide" href="{% url 'hosting:virtual_machines' %}">{% trans "OK" %}</a>
```

Comment:
We can remove the href parameter here? I guess, in all cases, we redirect user according to server's response?